docs/jegyzet/openapi_swagger/OpenAPI_Swagger.md

```diff
@@ -0,0 +1,222 @@
+# OpenAPI 3 és a Swagger
+
+## OpenAPI vs Swagger, mik is a különbségek?
+2017-ben az OpenAPI 3.0 megjelenése elég nagy mérföldkőnek számított. Ez volt az első hivatalos kiadás 2015 óta, amikor is a SmartBear Software az OpenAPI Intiative-nek ajándékozta a jogokat, illetve megtörtént a Swagger Specification -> OpenAPI Specification névváltás.
+
+A különbség legegyszerűbben így érthető meg: 
+- **OpenAPI** = specifikáció
+- **Swagger** = eszközök a specifikáció megvalósításához
+
+Az OpenAPI a specifikáció hivatalos neve. A specifikáció fejlesztését az OpenAPI Initiative segíti, amelyben több mint 30 szervezet vesz részt az IT világ különböző területeiről - köztük a Microsoft, a Google, az IBM és a CapitalOne. A Swagger eszközök fejlesztését vezető Smartbear Software szintén tagja az OpenAPI Initiative-nek, és segíti a specifikáció fejlődését.A Swagger eszköztár nyílt forráskódú, ingyenes és kereskedelmi eszközök keverékét tartalmazza, amelyek az API életciklus különböző szakaszaiban használhatók:
+
+ - **Swagger Editor**:  lehetővé teszi az OpenAPI-specifikációk YAML-ben történő szerkesztését a böngészőben, valamint a dokumentációk valós idejű előnézetét.
+ - **Swagger UI**: egy HTML, Javascript és CSS eszközökből álló gyűjtemény, amely dinamikusan gyönyörű dokumentációt generál egy OAS-kompatibilis API-ból.
+ - **Swagger Codegen**: Lehetővé teszi az API klienskönyvtárak (SDK generálása), szerver csonkok és dokumentáció automatikus generálását egy OpenAPI Spec alapján.
+ - **Swagger Parser**: Önálló könyvtár az OpenAPI definíciók Java-ból történő elemzésére.
+ - **Swagger Core**: Java-alapú könyvtárak az OpenAPI-definíciók létrehozásához, fogyasztásához és az azokkal való munkához.
+ - **Swagger Inspector** (ingyenes): API-tesztelő eszköz, amely lehetővé teszi az API-k validálását és OpenAPI-definíciók generálását egy meglévő API-ból.
+ - **SwaggerHub** (ingyenes és kereskedelmi): API-tervezés és dokumentáció, az OpenAPI-val dolgozó csapatok számára készült.
+ 
+Mélyedjünk el egy kicsit az OpenAPI 3.0 használatában.
+
+ ## Swagger 2.0
```

Comment:
Célja a dokumentumnak, hogy a swagger 2.0-át is bemutassa a 3.0 helyett? Én már csak a 3.0-ra koncentrálnék

Reply:
@tibitoth A Swagger 2.0 az OpenAPI 3.0 specifikáció eszközkészlete, a dokumentumomban külön dologként jelenik meg a kettő, annak ellenére, hogy a fejlesztői közösség sokszor a Swagger megnevezést használja az OpenAPI-ra.